docs/peggy tutorial

```diff
@@ -0,0 +1,47 @@
+First install dependencies:
+
+```
+INFURA_PROJECT_ID="abcdebeedabcdebeed"
+ETHEREUM_PRIVATE_KEY="0x00000000000000000000000000000000000000"
```

Comment:
Seems weird to have these env vars in a section called dependencies.

Reply:
We need to do a proper write up of the docs for peggy. This write up should cover all of these things instead of putting it all into a single doc. For now, this will do.